### PR TITLE
updated changelog for 1.0.1

### PR DESCRIPTION
**Pull request by shenda1**
PR is raised to update the changelog for 1.0.1

### CODE REVIEW

**Review by gokul-srivathsan — DISMISSED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,17 @@
-# v1.0.0
+# v1.0.1 (May 23, 2023)
+## Release Summary
+The release supports resources mentioned in the Features section for Dell PowerFlex.
+## Features
```
Comment (AnikaAgiwal2711) [inline CHANGELOG.md:4]: is this Features heading required??

**Review by gokul-srivathsan — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (AnikaAgiwal2711) [reply CHANGELOG.md:4]: Because I see nothing below that

Comment (AnikaAgiwal2711) [reply CHANGELOG.md:4]: Ok sorry my bad. Resources section is below that. Ignore this

Comment (shenda1) [reply CHANGELOG.md:4]: Is reources not visible?

**Review by AnikaAgiwal2711 — APPROVED** [review]

**Pull request merged**